### PR TITLE
A silly little change.

### PR DESCRIPTION
**Pull request by jshimkus-rh**
To demonstrate review vs comment diffs.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+A silly little change
```
Comment (jshimkus-rh) [inline LICENSE:1]: A line review of a silly little change.

**Review by jshimkus-rh — COMMENTED** [review]
The review level change.

**Comment by jshimkus-rh** [comment]
The review level change. From ghreview.

File: LICENSE (hunk 1 above)
Comment (jshimkus-rh) [inline LICENSE:1]: A line review of a silly little change.  From ghreview.

Comment (jshimkus) [inline LICENSE:1]: A line review from jshimkus, not jshimkus-rh.

**Review by jshimkus — COMMENTED** [review]
A pull-level comment from jshimkus.

**Pull request closed without merging**